convert2rhel/logger.py

```diff
@@ -37,6 +41,11 @@
 LOG_DIR = "/var/log/convert2rhel"
 
 
+class LogLevelCriticalNoExit:
+    level = 50
+    label = "CRITICAL"
```

Comment:
Wondering if we should identify this label as something else other than just "CRITICAL" to not confuse the user with this behavior… 

I'm thinking more like: "Oh, I ran convert2rhel in the past, and whenever I saw "CRITICAL", it meant that the execution would terminate, and now it just goes on". 

But I can't think of any special label that would go well alongside the critical name... Any ideas?

Reply:
I originally called it `CRITICAL_NO_EXIT` but later decided that the differences are probably below the level that the user will notice so it would just be confusing to the user.  Whether CRITICAL always exits or not is probably not something the user will notice for two reasons:

* We have always done a bunch of rollback stuff after the critical which obscures that we will rollback later.
* Since adding the Action framework, we perform all potential Actions before we exit due to a critical.  So it is not obvious that the exit is due to the critical rather than some other reported error.

So I think that this and `LogLevelCriticalWithExitt` (when we add it temporarily while we port the rest of the code over), should both report `CRITICAL` to the end user.

Does that make sense?

Reply:
Since it's a temporary thing, I see that it makes sense. Otherwise, I would say that this would look confusing. Even though the amount of rollback we do is quite high, and the message might get lost at some point, I can see someone looking for a critical message that does not interrupt the conversion. 

Myself would not expect that something critical happened, and we would be able to move forward...

Reply:
Maybe we can log something like "attempting to fix"